x/clock/genesis.go

```diff
@@ -44,18 +63,36 @@ func InitGenesis(
 	k keeper.Keeper,
 	data types.GenesisState,
 ) {
+	// Validate init contents
 	if err := ValidateGenesis(data); err != nil {
 		panic(err)
 	}
 
+	// Set params
 	if err := k.SetParams(ctx, data.Params); err != nil {
 		panic(err)
 	}
+
+	// Register unjailed contracts
+	for _, addr := range data.ContractAddresses {
```

Comment:
on InitGenesis there are no contracts on chain (InitGenesis is called at the start of the network, so no user interactions). See above comment related to this too

(same for jailed)

Reply:
Contracts removed from genesis